src/hooks/useDragAndDrop.ts

```diff
@@ -8,23 +8,30 @@ const DRAG_OVER_EVENT = 'dragover';
 const DRAG_LEAVE_EVENT = 'dragleave';
 const DROP_EVENT = 'drop';
 
+type DragAndDropParams = {
+    dropZone: React.MutableRefObject<HTMLDivElement | null>;
```

Comment:
this can only be ref of div element?

Reply:
@VickyStash it relates only to View in both cases and View renders as div, so I think yes